### PR TITLE
Mention default ServiceAccounts in the docs

### PR DESCRIPTION
**Pull request by qbarrand**
Add a paragraph about the label syncer for Pod Security Admission in OCP 4.12 and above.

/cc @mresvanis @ybettan

### CODE REVIEW

File: docs/mkdocs/documentation/deploy_kmod.md [hunk 1]
```diff
@@ -50,6 +50,45 @@ The reconciliation loop for `Module` runs the following steps:
...
+    - if the `Module` is created in the operator's namespace (`openshift-kmm` by default), then KMM will use its
+      default, powerful `ServiceAccounts` to run the DaemonSets;
+    - if the `Module` is created in any other namespace, then KMM will run the DaemonSets as the namespace's `default`
+      `ServiceAccount`, which cannot run privileged workload unless you manually allow it to use the `privileged` SCC.
+
+!!! warning "`openshift-kmm` is a trusted namespace"
+
+    When setting up RBAC permissions, keep in mind that any user or ServiceAccount creating a `Module` resource in the
+    `openshift-kmm` namespace will result in KMM automatically running privileged workload on potentially all nodes in
+    cluster.
```
Comment (ybettan) [inline docs/mkdocs/documentation/deploy_kmod.md:77]: If the user set RBAC then it will happen in any namespace no?!

Comment (qbarrand) [reply docs/mkdocs/documentation/deploy_kmod.md:77]: Correct. This warning reminds people that they don't need to explicitly set RBAC when creating a `Module` in `openshift-kmm` for the kmod to be loaded everywhere.

**Pull request merged**